packages/calcite-components/src/components/select/usage/Side-by-side.md

```diff
@@ -2,15 +2,15 @@ Separate the options side by side using a flex-box container.
 
 ```html
 <div class="demo-side-by-side">
-  <calcite-select width="half" label="half width">
-    <calcite-option>☕️</calcite-option>
-    <calcite-option>🍵</calcite-option>
-    <calcite-option selected>🍻</calcite-option>
+  <calcite-select width="half" label="Preferred Activities">
```

Comment:
Would `"Preferred Activities"` need to be singular here since only one `calcite-option` can be selected? If so, should the other example below on Music Genres also be modified?